cutil/cutil.go

```diff
@@ -93,7 +93,7 @@ func GenerateRandomNodeId() (string, error) {
 
 	bytes := make([]byte, 20, 20)
 	nodeIdString := ""
-	_, err := rand.Read(bytes)
+	_, err := crypto_rand.Read(bytes)
```

Comment:
what does this acturally changed? Isn't it still use `crypto/rand` library? @Yummikko